server/Controllers/BookingController.cs

```diff
@@ -21,6 +21,26 @@ public async Task<ActionResult<IEnumerable<BookingDto>>> GetAllBookings()
             var response = await _bookingService.GetAllBookings();
             return new OkObjectResult(response);
         }
+        [HttpGet("Bookings/{userId}")]
+        public async Task<ActionResult<IEnumerable<BookingDto>>> GetAllBookingsForPerson(int userId)
+        {
+            var response = await _bookingService.GetAllBookingsForPerson(userId);
+            return new OkObjectResult(response);
+        }
+
+        /// <summary>
+        /// Deletes a booking with the specified ID.
+        /// </summary>
+        /// <param name="id">The ID of the booking to delete.</param>
+        /// <returns>The result of the delete operation. 200 for succeed deleting, 404 for not found for the given id. and 500 for generic errors</returns>
+        [HttpDelete("Bookings/{id}")]
+        public async Task<ActionResult> DeleteBooking(int id)
```

Comment:
Vi burde kanskje ha en valideringssjekk for å validere at booking man ønsker å slette tilhører userId?

Reply:
den metode er å slette booking med bookingid. om at det trengs en delete allbooking som høre til en user id så må vi ha mer detaljer

Reply:
da blir det noe ekstra endpoint. og da skal vi diskutere om å slette booking med en gitt brukerid/ seatID som input.